inventory/sample-ha.aws.example.com.d/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,56 @@
+# 'hosting_infrastructure' is used to drive the correct behavior based
+# on the hosting infrastructure, cloud provider, etc. Valid values are:
+# - 'openstack'
+# - 'aws' (Coming Soon)
+# - 'azure' (Coming Soon)
+hosting_infrastructure: aws
+
+# AWS Provisioning Variables
+# use the "-e" option to set "aws_ssh_public_key"
+aws_image_name: <ec2 AMI>
```

Comment:
Can we fill in an example image name here - i.e.: the RHEL image in AWS?

Reply:
We could, but I think this could generate some issues/confusions to anyone using this repo as the AMI name changes on every Region. As an example this AMI in eu-west-1 and us-west-1: 

RHEL-7.4_HVM_GA-20170808-x86_64-2-Access2-GP2 - ami-2b9d6c52
RHEL-7.4_HVM_GA-20170808-x86_64-2-Access2-GP2 - ami-20eec540

Reply:
Main purpose for having an example is to help guide the user a bit more by having an example value that they may "recognize" by looking in the AWS console (even if it's somewhat different). Alternatively to providing an example value, we could use comments (which we need more of in the example inventories anyway)

Reply:
Sure, I have added an example and more comments to clarify this and other values in the inventory files